source/characterProcessing.py

```diff
@@ -343,11 +343,11 @@ def _loadSymbol(self, line):
 	def save(self, fileName=None):
 		"""Save symbol information to a file.
 		@param fileName: The name of the file to which to save symbol information,
-			C{None} to use the file name last passed to L{load} or L{save}.
+		        C{None} to use the file name last passed to L{load} or L{save}.
 		@type fileName: str
 		@raise IOError: If the file cannot be written.
 		@raise ValueError: If C{fileName} is C{None}
-			and L{load} or L{save} has not been called.
+		        and L{load} or L{save} has not been called.
```

Comment:
this went from tabs to spaces

Reply:
Are we sure that Ruff isn't doing bad stuff to our doc strings? I haven't done anything else than just Ruff formatting and the default pre-commit setup

Reply:
From what I can tell Ruff format didn't do this when it was applied `ruff format` to the whole codebase. Is it possible your IDE is doing this?

Reply:
Possibly, but my VS Code is set to use Ruff formatting all over the place. I'll try to track this down